pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -214,18 +215,27 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			fmt.Fprintln(out, "Errors:")
 			for _, marker := range errorMarkers {
 				fmt.Fprintln(out, indent+"* "+marker.Message)
+				if d.Suggest && len(marker.Suggestion) > 0 {
+					fmt.Fprintln(out, indent+marker.Suggestion.String())
```

Comment:
I think we might want a better indication that this is for a suggestion.

@jwforres @fabianofranz Ideas?  I might come up with something ugly like: 

```
* this is the error message
++ this is the suggestion
```